typings/index.d.ts

```diff
@@ -1,12 +1,12 @@
 declare enum ChannelType {
-	text,
-	dm,
-	voice,
-	group,
-	category,
-	news,
-	store,
-	unknown
+	TEXT = 'text',
+	DM = 'dm',
+	VOICE = 'voice',
+	GROUP = 'group',
+	CATEGORY = 'category',
+	NEWS = 'news',
+	STORE = 'store',
+	UNKNOWN = 'unknown'
```

Comment:
TypeScript standard it to use pascal-case for enums ([source: TypeScript docs](typescriptlang.org/docs/handbook/enums.html#string-enums)). Enums are inherently constant, so it doesn't really matter that we use uppercase here.

Reply:
Your link seems to be invalid, could you toss me the proper one? I cannot seem to find this standard/convention you are talking about.

Reply:
Sorry, here's the correct link:

https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

You can see in their example codeblocks from that page that they are using Pascal case.

Reply:
Despite it commonly being in official documentation, it is never stated that pascal case is the standard.

In fact, there's a few cases on that very page that use full upppercase.

I don't feel like I'm breaking a convention here so unless anyone else has something to say about this I'm leaving it this way.